03/app.js

```diff
@@ -11,4 +11,16 @@ const buttonSettings = {
         color: '#444'
     },
     text: 'Click me!',
-}
\ No newline at end of file
+}
+
+const sectionEl = document.querySelector('section');
+const buttonEl = document.createElement('button');
+buttonEl.setAttribute('class', buttonSettings.attr.className);
+buttonEl.setAttribute('title', buttonSettings.attr.title);
+buttonEl.style.border = buttonSettings.css.border;
+buttonEl.style.padding = buttonSettings.css.padding;
+buttonEl.style.color = buttonSettings.css.color;
+buttonEl.textContent = buttonSettings.text;
```

Comment:
Spróbuj to zrobić dzięki pętli. Co jeśli będzie 100 pozycji w obiekcie? Wykonanie wtedy tego ręcznie będzie bardzo pracochłonne.